.github/workflows/ci-tests.yml

```diff
@@ -5,6 +5,11 @@ on:
     branches:
       - master
       - release-**
+    types:
```

Comment:
adding `ready_for_review` along with default trigger types
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request